libs/utils/fuzzing/src/FilterFuzz.cc

```diff
@@ -0,0 +1,23 @@
+#include <celix_filter.h>
+#include <celix_err.h>
+#include <cstdint>
+#include <cstdlib>
+#include <cstring>
+
+int filterParseFuzzOneInput(const uint8_t* data, size_t size) {
+    char* buffer = static_cast<char*>(malloc(size + 1));
+    if (buffer == nullptr) {
+        return 0;
+    }
+    memcpy(buffer, data, size);
+    buffer[size] = '\0';
+
+    celix_filter_t* filter = celix_filter_create(buffer);
```

Comment:
`celix_filter_match` seems another interesting fuzzing target.

Reply:
Look into this, but I think this is not really feasible. The fuzz testing is focused on fuzzing a single input buffer and for filter match you need two (properties and a filter).